__tests__/app.test.js

```diff
@@ -1,87 +1,199 @@
-    })
-   })
+      return request(app)
+        .get("/api/topic")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Not Found!");
+        });
+    });
+  });
 });
 
 describe("GET /api/articles/:article_id", () => {
-    describe("STATUS 200", () => {
-        test("responds with status 200 and returns relevant article", () => {
-            return request(app)
-            .get('/api/articles/1')
-            .expect(200)
-            .then((res) => {
-                const date = new Date(1594329060000).toJSON();
-              expect(res.body.article).toEqual({
-                username: 'butter_bridge',
-                article_id: 1,
-                title: "Living in the shadow of a great man",
-                topic: "mitch",
-                body: "I find this existence challenging",
-                created_at: date,
-                votes: 100
-              });
-            });
-        })
-    })
-    describe("STATUS 400s", () => {
-        test("Status 400 - invalid id", () => {
-            return request(app)
-            .get('/api/articles/invalidId')
-            .expect(400)
-            .then((res) => {
-            expect(res.body.msg).toBe('Invalid id');
-            });
-        })
-        test("STATUS 404 - article doesn't exist", () => {
-            return request(app).get('/api/articles/999').expect(404).then((res) => {
-            expect(res.body.msg).toBe("Article doesn't exist");
-             });
-        })
-    })
-})
\ No newline at end of file
+  describe("STATUS 200", () => {
+    test("responds with status 200 and returns relevant article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          const date = new Date(1594329060000).toJSON();
+          expect(res.body.article).toEqual({
+            username: "butter_bridge",
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            body: "I find this existence challenging",
+            created_at: date,
+            votes: 100,
+          });
+        });
+    });
+  });
+  describe("STATUS 400s", () => {
+    test("Status 400 - invalid id", () => {
+      return request(app)
+        .get("/api/articles/invalidId")
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toBe("Invalid input data type");
+        });
+    });
+    test("STATUS 404 - article doesn't exist", () => {
+      return request(app)
+        .get("/api/articles/999")
+        .expect(404)
+        .then((res) => {
+          expect(res.body.msg).toBe("Article not found!");
+        });
+    });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  describe("STATUS 200", () => {
+    test("Responds with 200 and returns relevant article with incremented votes", () => {
+      const newVote = { inc_votes: 1 };
+      const date = new Date(1594329060000).toJSON();
```

Comment:
the purpose of creating a date as part of this test is unclear to me

Reply:
@alexdevdavis  If I copy the long number in the object I provide test fails as it takes those numbers literally